error: Error fatal
warning: Mensaje de Warning
error: Error fatal
warning: Mensaje de Warning
error: Error fatal
warning: Mensaje de Warning
error: Error al obtener productos getaddrinfo ENOTFOUND ac-gimvjmn-shard-00-00.sasvmwp.mongodb.net {"reason":{"commonWireVersion":0,"compatible":true,"heartbeatFrequencyMS":10000,"localThresholdMS":15,"logicalSessionTimeoutMinutes":null,"maxElectionId":"7fffffff0000000000000022","maxSetVersion":3,"servers":{},"setName":"atlas-tea0ch-shard-0","stale":false,"type":"ReplicaSetNoPrimary"},"stack":"MongoServerSelectionError: getaddrinfo ENOTFOUND ac-gimvjmn-shard-00-00.sasvmwp.mongodb.net\n    at EventTarget.<anonymous> (C:\\Users\\Evelin\\Desktop\\desafio-complementario-3\\node_modules\\mongodb\\lib\\sdam\\topology.js:289:34)\n    at [nodejs.internal.kHybridDispatch] (node:internal/event_target:814:20)\n    at EventTarget.dispatchEvent (node:internal/event_target:749:26)\n    at abortSignal (node:internal/abort_controller:371:10)\n    at TimeoutController.abort (node:internal/abort_controller:393:5)\n    at Timeout.<anonymous> (C:\\Users\\Evelin\\Desktop\\desafio-complementario-3\\node_modules\\mongodb\\lib\\utils.js:976:92)\n    at listOnTimeout (node:internal/timers:573:17)\n    at process.processTimers (node:internal/timers:514:7)"}
error: Error al procesar la compra: Error al enviar correo electrónico {"stack":"Error: Error al enviar correo electrónico\n    at EmailManager.purchaseEmail (C:\\Users\\Evelin\\Desktop\\desafio-complementario-3\\src\\services\\email.js:30:19)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async finalizePurchase (C:\\Users\\Evelin\\Desktop\\desafio-complementario-3\\src\\controllers\\cart.controllers.js:164:12)"}
error: Error al obtener los detalles del producto: Cannot read properties of null (reading 'cart') {"stack":"TypeError: Cannot read properties of null (reading 'cart')\n    at renderProductDetails (C:\\Users\\Evelin\\Desktop\\desafio-11-backend\\src\\controllers\\views.controllers.js:73:24)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)"}
error: Error en el registro de usuario: Error al intentar crear un usuario {"causa":" Los datos estan incompletos o no son válidos. \n    Necesitamos recibir los siguientes datos: \n    - Nombre: String, pero recibimos undefined\n    - Apellido: String, pero recibimos undefined\n    - Email: String, recibimos undefined\n    -Password: recibimos undefined\n    -Edad: recibimos undefined\n    ","code":2,"name":"Error","stack":"Error: Error al intentar crear un usuario\n    at CustomErrors.createError (C:\\Users\\Evelin\\Desktop\\desafio-11-backend\\src\\services\\errors\\custom-error.js:3:23)\n    at register (C:\\Users\\Evelin\\Desktop\\desafio-11-backend\\src\\controllers\\user.controllers.js:21:28)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)"}
error: Error en el registro de usuario: Error al intentar crear un usuario {"causa":" Los datos estan incompletos o no son válidos. \n    Necesitamos recibir los siguientes datos: \n    - Nombre: String, pero recibimos undefined\n    - Apellido: String, pero recibimos undefined\n    - Email: String, recibimos undefined\n    -Password: recibimos undefined\n    -Edad: recibimos undefined\n    ","code":2,"name":"Error","stack":"Error: Error al intentar crear un usuario\n    at CustomErrors.createError (C:\\Users\\Evelin\\Desktop\\desafio-11-backend\\src\\services\\errors\\custom-error.js:3:23)\n    at register (C:\\Users\\Evelin\\Desktop\\desafio-11-backend\\src\\controllers\\user.controllers.js:21:28)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)"}
error: Error en el registro de usuario: Error al intentar crear un usuario {"causa":" Los datos estan incompletos o no son válidos. \n    Necesitamos recibir los siguientes datos: \n    - Nombre: String, pero recibimos undefined\n    - Apellido: String, pero recibimos undefined\n    - Email: String, recibimos undefined\n    -Password: recibimos undefined\n    -Edad: recibimos undefined\n    ","code":2,"name":"Error","stack":"Error: Error al intentar crear un usuario\n    at CustomErrors.createError (C:\\Users\\Evelin\\Desktop\\desafio-11-backend\\src\\services\\errors\\custom-error.js:3:23)\n    at register (C:\\Users\\Evelin\\Desktop\\desafio-11-backend\\src\\controllers\\user.controllers.js:21:28)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)"}
error: Error en el registro de usuario: Error al intentar crear un usuario {"causa":" Los datos estan incompletos o no son válidos. \n    Necesitamos recibir los siguientes datos: \n    - Nombre: String, pero recibimos undefined\n    - Apellido: String, pero recibimos undefined\n    - Email: String, recibimos undefined\n    -Password: recibimos undefined\n    -Edad: recibimos undefined\n    ","code":2,"name":"Error","stack":"Error: Error al intentar crear un usuario\n    at CustomErrors.createError (C:\\Users\\Evelin\\Desktop\\desafio-11-backend\\src\\services\\errors\\custom-error.js:3:23)\n    at register (C:\\Users\\Evelin\\Desktop\\desafio-11-backend\\src\\controllers\\user.controllers.js:21:28)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)"}
error: Error en el registro de usuario: Error al intentar crear un usuario {"causa":" Los datos estan incompletos o no son válidos. \n    Necesitamos recibir los siguientes datos: \n    - Nombre: String, pero recibimos undefined\n    - Apellido: String, pero recibimos undefined\n    - Email: String, recibimos undefined\n    -Password: recibimos undefined\n    -Edad: recibimos undefined\n    ","code":2,"name":"Error","stack":"Error: Error al intentar crear un usuario\n    at CustomErrors.createError (C:\\Users\\Evelin\\Desktop\\desafio-11-backend\\src\\services\\errors\\custom-error.js:3:23)\n    at register (C:\\Users\\Evelin\\Desktop\\desafio-11-backend\\src\\controllers\\user.controllers.js:21:28)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)"}
error: Error en el registro de usuario: Error al intentar crear un usuario {"causa":" Los datos estan incompletos o no son válidos. \n    Necesitamos recibir los siguientes datos: \n    - Nombre: String, pero recibimos undefined\n    - Apellido: String, pero recibimos undefined\n    - Email: String, recibimos undefined\n    -Password: recibimos undefined\n    -Edad: recibimos undefined\n    ","code":2,"name":"Error","stack":"Error: Error al intentar crear un usuario\n    at CustomErrors.createError (C:\\Users\\Evelin\\Desktop\\desafio-11-backend\\src\\services\\errors\\custom-error.js:3:23)\n    at register (C:\\Users\\Evelin\\Desktop\\desafio-11-backend\\src\\controllers\\user.controllers.js:21:28)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)"}
error: Error al obtener el producto por ID: Error {"stack":"Error: Error\n    at ProductRepository.getProductsById (C:\\Users\\Evelin\\Desktop\\desafio-11-backend\\src\\repositories\\product.repository.js:103:13)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async getProductsById (C:\\Users\\Evelin\\Desktop\\desafio-11-backend\\src\\controllers\\product.controllers.js:37:28)"}
error: Error al obtener el producto por ID: Error {"stack":"Error: Error\n    at ProductRepository.getProductsById (C:\\Users\\Evelin\\Desktop\\desafio-11-backend\\src\\repositories\\product.repository.js:103:13)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async getProductsById (C:\\Users\\Evelin\\Desktop\\desafio-11-backend\\src\\controllers\\product.controllers.js:37:28)"}
error: Error al obtener el producto por ID: Error {"stack":"Error: Error\n    at ProductRepository.getProductsById (C:\\Users\\Evelin\\Desktop\\desafio-11-backend\\src\\repositories\\product.repository.js:103:13)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async getProductsById (C:\\Users\\Evelin\\Desktop\\desafio-11-backend\\src\\controllers\\product.controllers.js:37:28)"}
error: Error al obtener el producto por ID: Error {"stack":"Error: Error\n    at ProductRepository.getProductsById (C:\\Users\\Evelin\\Desktop\\desafio-11-backend\\src\\repositories\\product.repository.js:103:13)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async getProductsById (C:\\Users\\Evelin\\Desktop\\desafio-11-backend\\src\\controllers\\product.controllers.js:37:28)"}
error: Error al obtener el producto por ID: Error {"stack":"Error: Error\n    at ProductRepository.getProductsById (C:\\Users\\Evelin\\Desktop\\desafio-11-backend\\src\\repositories\\product.repository.js:103:13)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async getProductsById (C:\\Users\\Evelin\\Desktop\\desafio-11-backend\\src\\controllers\\product.controllers.js:37:28)"}
error: Error al obtener el producto por ID: Error {"stack":"Error: Error\n    at ProductRepository.getProductsById (C:\\Users\\Evelin\\Desktop\\desafio-11-backend\\src\\repositories\\product.repository.js:103:13)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async getProductsById (C:\\Users\\Evelin\\Desktop\\desafio-11-backend\\src\\controllers\\product.controllers.js:37:28)"}
error: Error al obtener el producto por ID: Error {"stack":"Error: Error\n    at ProductRepository.getProductsById (C:\\Users\\Evelin\\Desktop\\desafio-11-backend\\src\\repositories\\product.repository.js:103:13)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async getProductsById (C:\\Users\\Evelin\\Desktop\\desafio-11-backend\\src\\controllers\\product.controllers.js:37:28)"}
error: Error al obtener el producto por ID: Error {"stack":"Error: Error\n    at ProductRepository.getProductsById (C:\\Users\\Evelin\\Desktop\\desafio-11-backend\\src\\repositories\\product.repository.js:103:13)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async getProductsById (C:\\Users\\Evelin\\Desktop\\desafio-11-backend\\src\\controllers\\product.controllers.js:37:28)"}
error: Error al obtener el producto por ID: Error {"stack":"Error: Error\n    at ProductRepository.getProductsById (C:\\Users\\Evelin\\Desktop\\desafio-11-backend\\src\\repositories\\product.repository.js:103:13)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async getProductsById (C:\\Users\\Evelin\\Desktop\\desafio-11-backend\\src\\controllers\\product.controllers.js:37:28)"}
error: Error al obtener el producto por ID: Error {"stack":"Error: Error\n    at ProductRepository.getProductsById (C:\\Users\\Evelin\\Desktop\\desafio-11-backend\\src\\repositories\\product.repository.js:103:13)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async getProductsById (C:\\Users\\Evelin\\Desktop\\desafio-11-backend\\src\\controllers\\product.controllers.js:37:28)"}
error: Error al obtener el producto por ID: Error {"stack":"Error: Error\n    at ProductRepository.getProductsById (C:\\Users\\Evelin\\Desktop\\desafio-11-backend\\src\\repositories\\product.repository.js:103:13)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async getProductsById (C:\\Users\\Evelin\\Desktop\\desafio-11-backend\\src\\controllers\\product.controllers.js:37:28)"}
error: Error al obtener el producto por ID: Error {"stack":"Error: Error\n    at ProductRepository.getProductsById (C:\\Users\\Evelin\\Desktop\\desafio-11-backend\\src\\repositories\\product.repository.js:103:13)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async getProductsById (C:\\Users\\Evelin\\Desktop\\desafio-11-backend\\src\\controllers\\product.controllers.js:37:28)"}
error: Error al eliminar el producto: Error {"stack":"Error: Error\n    at ProductRepository.deleteProduct (C:\\Users\\Evelin\\Desktop\\desafio-11-backend\\src\\repositories\\product.repository.js:131:13)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async deleteProduct (C:\\Users\\Evelin\\Desktop\\desafio-11-backend\\src\\controllers\\product.controllers.js:69:26)"}
error: Error al eliminar el producto: Error {"stack":"Error: Error\n    at ProductRepository.deleteProduct (C:\\Users\\Evelin\\Desktop\\desafio-11-backend\\src\\repositories\\product.repository.js:131:13)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async deleteProduct (C:\\Users\\Evelin\\Desktop\\desafio-11-backend\\src\\controllers\\product.controllers.js:69:26)"}
error: Error al obtener el producto por ID: Error {"stack":"Error: Error\n    at ProductRepository.getProductsById (C:\\Users\\Evelin\\Desktop\\desafio-11-backend\\src\\repositories\\product.repository.js:103:13)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async getProductsById (C:\\Users\\Evelin\\Desktop\\desafio-11-backend\\src\\controllers\\product.controllers.js:37:28)"}
error: Error al obtener el producto por ID: Error {"stack":"Error: Error\n    at ProductRepository.getProductsById (C:\\Users\\Evelin\\Desktop\\desafio-11-backend\\src\\repositories\\product.repository.js:103:13)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async getProductsById (C:\\Users\\Evelin\\Desktop\\desafio-11-backend\\src\\controllers\\product.controllers.js:37:28)"}
error: Error al eliminar el producto: Error {"stack":"Error: Error\n    at ProductRepository.deleteProduct (C:\\Users\\Evelin\\Desktop\\desafio-11-backend\\src\\repositories\\product.repository.js:131:13)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async deleteProduct (C:\\Users\\Evelin\\Desktop\\desafio-11-backend\\src\\controllers\\product.controllers.js:69:26)"}
error: Error al eliminar el producto: Error {"stack":"Error: Error\n    at ProductRepository.deleteProduct (C:\\Users\\Evelin\\Desktop\\desafio-11-backend\\src\\repositories\\product.repository.js:131:13)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async deleteProduct (C:\\Users\\Evelin\\Desktop\\desafio-11-backend\\src\\controllers\\product.controllers.js:69:26)"}
error: Error al obtener el producto por ID: Error {"stack":"Error: Error\n    at ProductRepository.getProductsById (C:\\Users\\Evelin\\Desktop\\desafio-11-backend\\src\\repositories\\product.repository.js:103:13)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async getProductsById (C:\\Users\\Evelin\\Desktop\\desafio-11-backend\\src\\controllers\\product.controllers.js:37:28)"}
error: Error al obtener el producto por ID: Error {"stack":"Error: Error\n    at ProductRepository.getProductsById (C:\\Users\\Evelin\\Desktop\\desafio-11-backend\\src\\repositories\\product.repository.js:103:13)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async getProductsById (C:\\Users\\Evelin\\Desktop\\desafio-11-backend\\src\\controllers\\product.controllers.js:37:28)"}
error: Error al obtener el producto por ID: Error {"stack":"Error: Error\n    at ProductRepository.getProductsById (C:\\Users\\Evelin\\Desktop\\desafio-11-backend\\src\\repositories\\product.repository.js:103:13)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async getProductsById (C:\\Users\\Evelin\\Desktop\\desafio-11-backend\\src\\controllers\\product.controllers.js:37:28)"}
error: Error al obtener el producto por ID: Error {"stack":"Error: Error\n    at ProductRepository.getProductsById (C:\\Users\\Evelin\\Desktop\\desafio-11-backend\\src\\repositories\\product.repository.js:103:13)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async getProductsById (C:\\Users\\Evelin\\Desktop\\desafio-11-backend\\src\\controllers\\product.controllers.js:37:28)"}
error: Error al eliminar el producto: Error {"stack":"Error: Error\n    at ProductRepository.deleteProduct (C:\\Users\\Evelin\\Desktop\\desafio-11-backend\\src\\repositories\\product.repository.js:131:13)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async deleteProduct (C:\\Users\\Evelin\\Desktop\\desafio-11-backend\\src\\controllers\\product.controllers.js:69:26)"}
error: Error al eliminar el producto: Error {"stack":"Error: Error\n    at ProductRepository.deleteProduct (C:\\Users\\Evelin\\Desktop\\desafio-11-backend\\src\\repositories\\product.repository.js:131:13)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async deleteProduct (C:\\Users\\Evelin\\Desktop\\desafio-11-backend\\src\\controllers\\product.controllers.js:69:26)"}
error: Error al obtener el producto por ID: Error {"stack":"Error: Error\n    at ProductRepository.getProductsById (C:\\Users\\Evelin\\Desktop\\desafio-11-backend\\src\\repositories\\product.repository.js:103:13)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async getProductsById (C:\\Users\\Evelin\\Desktop\\desafio-11-backend\\src\\controllers\\product.controllers.js:37:28)"}
error: Error al obtener el producto por ID: Error {"stack":"Error: Error\n    at ProductRepository.getProductsById (C:\\Users\\Evelin\\Desktop\\desafio-11-backend\\src\\repositories\\product.repository.js:103:13)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async getProductsById (C:\\Users\\Evelin\\Desktop\\desafio-11-backend\\src\\controllers\\product.controllers.js:37:28)"}
error: Error al obtener el producto por ID: Error {"stack":"Error: Error\n    at ProductRepository.getProductsById (C:\\Users\\Evelin\\Desktop\\desafio-11-backend\\src\\repositories\\product.repository.js:103:13)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async getProductsById (C:\\Users\\Evelin\\Desktop\\desafio-11-backend\\src\\controllers\\product.controllers.js:37:28)"}
error: Error al obtener el producto por ID: Error {"stack":"Error: Error\n    at ProductRepository.getProductsById (C:\\Users\\Evelin\\Desktop\\desafio-11-backend\\src\\repositories\\product.repository.js:103:13)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async getProductsById (C:\\Users\\Evelin\\Desktop\\desafio-11-backend\\src\\controllers\\product.controllers.js:37:28)"}
error: Error al obtener el producto por ID: Error {"stack":"Error: Error\n    at ProductRepository.getProductsById (C:\\Users\\Evelin\\Desktop\\desafio-11-backend\\src\\repositories\\product.repository.js:103:13)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async getProductsById (C:\\Users\\Evelin\\Desktop\\desafio-11-backend\\src\\controllers\\product.controllers.js:37:28)"}
error: Error al obtener el producto por ID: Error {"stack":"Error: Error\n    at ProductRepository.getProductsById (C:\\Users\\Evelin\\Desktop\\desafio-11-backend\\src\\repositories\\product.repository.js:103:13)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async getProductsById (C:\\Users\\Evelin\\Desktop\\desafio-11-backend\\src\\controllers\\product.controllers.js:37:28)"}
error: Error al obtener el producto por ID: Error {"stack":"Error: Error\n    at ProductRepository.getProductsById (C:\\Users\\Evelin\\Desktop\\desafio-11-backend\\src\\repositories\\product.repository.js:103:13)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async getProductsById (C:\\Users\\Evelin\\Desktop\\desafio-11-backend\\src\\controllers\\product.controllers.js:37:28)"}
error: Error al obtener el producto por ID: Error {"stack":"Error: Error\n    at ProductRepository.getProductsById (C:\\Users\\Evelin\\Desktop\\desafio-11-backend\\src\\repositories\\product.repository.js:103:13)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async getProductsById (C:\\Users\\Evelin\\Desktop\\desafio-11-backend\\src\\controllers\\product.controllers.js:37:28)"}
error: Error al obtener el producto por ID: Error {"stack":"Error: Error\n    at ProductRepository.getProductsById (C:\\Users\\Evelin\\Desktop\\desafio-11-backend\\src\\repositories\\product.repository.js:103:13)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async getProductsById (C:\\Users\\Evelin\\Desktop\\desafio-11-backend\\src\\controllers\\product.controllers.js:37:28)"}
error: Error al obtener el producto por ID: Error {"stack":"Error: Error\n    at ProductRepository.getProductsById (C:\\Users\\Evelin\\Desktop\\desafio-11-backend\\src\\repositories\\product.repository.js:103:13)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async getProductsById (C:\\Users\\Evelin\\Desktop\\desafio-11-backend\\src\\controllers\\product.controllers.js:37:28)"}
error: Error al obtener el producto por ID: Error {"stack":"Error: Error\n    at ProductRepository.getProductsById (C:\\Users\\Evelin\\Desktop\\desafio-11-backend\\src\\repositories\\product.repository.js:103:13)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async getProductsById (C:\\Users\\Evelin\\Desktop\\desafio-11-backend\\src\\controllers\\product.controllers.js:37:28)"}
error: Error al obtener el producto por ID: Error {"stack":"Error: Error\n    at ProductRepository.getProductsById (C:\\Users\\Evelin\\Desktop\\desafio-11-backend\\src\\repositories\\product.repository.js:103:13)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async getProductsById (C:\\Users\\Evelin\\Desktop\\desafio-11-backend\\src\\controllers\\product.controllers.js:37:28)"}
error: Error al actualizar el producto: Error {"stack":"Error: Error\n    at ProductRepository.updateProduct (C:\\Users\\Evelin\\Desktop\\desafio-11-backend\\src\\repositories\\product.repository.js:117:13)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async updateProduct (C:\\Users\\Evelin\\Desktop\\desafio-11-backend\\src\\controllers\\product.controllers.js:56:28)"}
error: Error al actualizar el producto: Error {"stack":"Error: Error\n    at ProductRepository.updateProduct (C:\\Users\\Evelin\\Desktop\\desafio-11-backend\\src\\repositories\\product.repository.js:117:13)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async updateProduct (C:\\Users\\Evelin\\Desktop\\desafio-11-backend\\src\\controllers\\product.controllers.js:56:28)"}
error: Error al actualizar el producto: Error {"stack":"Error: Error\n    at ProductRepository.updateProduct (C:\\Users\\Evelin\\Desktop\\desafio-11-backend\\src\\repositories\\product.repository.js:117:13)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async updateProduct (C:\\Users\\Evelin\\Desktop\\desafio-11-backend\\src\\controllers\\product.controllers.js:56:28)"}
error: Error al actualizar el producto: Error {"stack":"Error: Error\n    at ProductRepository.updateProduct (C:\\Users\\Evelin\\Desktop\\desafio-11-backend\\src\\repositories\\product.repository.js:117:13)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async updateProduct (C:\\Users\\Evelin\\Desktop\\desafio-11-backend\\src\\controllers\\product.controllers.js:56:28)"}
error: Error al actualizar el producto: Error {"stack":"Error: Error\n    at ProductRepository.updateProduct (C:\\Users\\Evelin\\Desktop\\desafio-11-backend\\src\\repositories\\product.repository.js:117:13)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async updateProduct (C:\\Users\\Evelin\\Desktop\\desafio-11-backend\\src\\controllers\\product.controllers.js:56:28)"}
error: Error al obtener el producto por ID: Error {"stack":"Error: Error\n    at ProductRepository.getProductsById (C:\\Users\\Evelin\\Desktop\\desafio-11-backend\\src\\repositories\\product.repository.js:103:13)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async getProductsById (C:\\Users\\Evelin\\Desktop\\desafio-11-backend\\src\\controllers\\product.controllers.js:37:28)"}
error: Error al obtener el producto por ID: Error {"stack":"Error: Error\n    at ProductRepository.getProductsById (C:\\Users\\Evelin\\Desktop\\desafio-11-backend\\src\\repositories\\product.repository.js:103:13)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async getProductsById (C:\\Users\\Evelin\\Desktop\\desafio-11-backend\\src\\controllers\\product.controllers.js:37:28)"}
error: Error en el registro de usuario: Error al intentar crear un usuario {"causa":" Los datos estan incompletos o no son válidos. \n    Necesitamos recibir los siguientes datos: \n    - Nombre: String, pero recibimos undefined\n    - Apellido: String, pero recibimos undefined\n    - Email: String, recibimos undefined\n    -Password: recibimos undefined\n    -Edad: recibimos undefined\n    ","code":2,"name":"Error","stack":"Error: Error al intentar crear un usuario\n    at CustomErrors.createError (C:\\Users\\Evelin\\Desktop\\desafio-11-backend\\src\\services\\errors\\custom-error.js:3:23)\n    at register (C:\\Users\\Evelin\\Desktop\\desafio-11-backend\\src\\controllers\\user.controllers.js:21:28)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)"}
error: Error en el registro de usuario: Error al intentar crear un usuario {"causa":" Los datos estan incompletos o no son válidos. \n    Necesitamos recibir los siguientes datos: \n    - Nombre: String, pero recibimos undefined\n    - Apellido: String, pero recibimos undefined\n    - Email: String, recibimos undefined\n    -Password: recibimos undefined\n    -Edad: recibimos undefined\n    ","code":2,"name":"Error","stack":"Error: Error al intentar crear un usuario\n    at CustomErrors.createError (C:\\Users\\Evelin\\Desktop\\practica-integradora-4-back\\src\\services\\errors\\custom-error.js:3:23)\n    at register (C:\\Users\\Evelin\\Desktop\\practica-integradora-4-back\\src\\controllers\\user.controllers.js:21:28)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)"}
error: Error al conectar a MongoDB: connect ETIMEDOUT 18.229.182.75:27017 {"reason":{"commonWireVersion":0,"compatible":true,"heartbeatFrequencyMS":10000,"localThresholdMS":15,"logicalSessionTimeoutMinutes":30,"maxElectionId":null,"maxSetVersion":null,"servers":{},"setName":"atlas-tea0ch-shard-0","stale":false,"type":"ReplicaSetNoPrimary"},"stack":"MongooseServerSelectionError: connect ETIMEDOUT 18.229.182.75:27017\n    at _handleConnectionErrors (C:\\Users\\Evelin\\Desktop\\practica-integradora-4-back\\node_modules\\mongoose\\lib\\connection.js:875:11)\n    at NativeConnection.openUri (C:\\Users\\Evelin\\Desktop\\practica-integradora-4-back\\node_modules\\mongoose\\lib\\connection.js:826:11)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)"}
error: Error al conectar a MongoDB: connect ETIMEDOUT 52.67.72.115:27017 {"connectionGeneration":0,"stack":"MongoNetworkError: connect ETIMEDOUT 52.67.72.115:27017\n    at connectionFailureError (C:\\Users\\Evelin\\Desktop\\practica-integradora-4-back\\node_modules\\mongodb\\lib\\cmap\\connect.js:353:20)\n    at TLSSocket.<anonymous> (C:\\Users\\Evelin\\Desktop\\practica-integradora-4-back\\node_modules\\mongodb\\lib\\cmap\\connect.js:268:44)\n    at Object.onceWrapper (node:events:629:26)\n    at TLSSocket.emit (node:events:514:28)\n    at emitErrorNT (node:internal/streams/destroy:151:8)\n    at emitErrorCloseNT (node:internal/streams/destroy:116:3)\n    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)"}
error: Error en el inicio de sesión: Operation `users.findOne()` buffering timed out after 10000ms {"stack":"MongooseError: Operation `users.findOne()` buffering timed out after 10000ms\n    at Timeout.<anonymous> (C:\\Users\\Evelin\\Desktop\\practica-integradora-4-back\\node_modules\\mongoose\\lib\\drivers\\node-mongodb-native\\collection.js:185:23)\n    at listOnTimeout (node:internal/timers:573:17)\n    at process.processTimers (node:internal/timers:514:7)"}
error: Error al conectar a MongoDB: connect ETIMEDOUT 52.67.72.115:27017 {"connectionGeneration":1,"stack":"MongoNetworkError: connect ETIMEDOUT 52.67.72.115:27017\n    at connectionFailureError (C:\\Users\\Evelin\\Desktop\\practica-integradora-4-back\\node_modules\\mongodb\\lib\\cmap\\connect.js:353:20)\n    at TLSSocket.<anonymous> (C:\\Users\\Evelin\\Desktop\\practica-integradora-4-back\\node_modules\\mongodb\\lib\\cmap\\connect.js:268:44)\n    at Object.onceWrapper (node:events:629:26)\n    at TLSSocket.emit (node:events:514:28)\n    at emitErrorNT (node:internal/streams/destroy:151:8)\n    at emitErrorCloseNT (node:internal/streams/destroy:116:3)\n    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)"}
error: Error al conectar a MongoDB: connect ETIMEDOUT 52.67.72.115:27017 {"connectionGeneration":1,"stack":"MongoNetworkError: connect ETIMEDOUT 52.67.72.115:27017\n    at connectionFailureError (C:\\Users\\Evelin\\Desktop\\practica-integradora-4-back\\node_modules\\mongodb\\lib\\cmap\\connect.js:353:20)\n    at TLSSocket.<anonymous> (C:\\Users\\Evelin\\Desktop\\practica-integradora-4-back\\node_modules\\mongodb\\lib\\cmap\\connect.js:268:44)\n    at Object.onceWrapper (node:events:629:26)\n    at TLSSocket.emit (node:events:514:28)\n    at emitErrorNT (node:internal/streams/destroy:151:8)\n    at emitErrorCloseNT (node:internal/streams/destroy:116:3)\n    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)"}
error: Error al conectar a MongoDB: connect ETIMEDOUT 52.67.72.115:27017 {"connectionGeneration":1,"stack":"MongoNetworkError: connect ETIMEDOUT 52.67.72.115:27017\n    at connectionFailureError (C:\\Users\\Evelin\\Desktop\\practica-integradora-4-back\\node_modules\\mongodb\\lib\\cmap\\connect.js:353:20)\n    at TLSSocket.<anonymous> (C:\\Users\\Evelin\\Desktop\\practica-integradora-4-back\\node_modules\\mongodb\\lib\\cmap\\connect.js:268:44)\n    at Object.onceWrapper (node:events:629:26)\n    at TLSSocket.emit (node:events:514:28)\n    at emitErrorNT (node:internal/streams/destroy:151:8)\n    at emitErrorCloseNT (node:internal/streams/destroy:116:3)\n    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)"}
error: Error al conectar a MongoDB: connect ETIMEDOUT 52.67.72.115:27017 {"reason":{"commonWireVersion":0,"compatible":true,"heartbeatFrequencyMS":10000,"localThresholdMS":15,"logicalSessionTimeoutMinutes":30,"maxElectionId":null,"maxSetVersion":null,"servers":{},"setName":"atlas-tea0ch-shard-0","stale":false,"type":"ReplicaSetNoPrimary"},"stack":"MongooseServerSelectionError: connect ETIMEDOUT 52.67.72.115:27017\n    at _handleConnectionErrors (C:\\Users\\Evelin\\Desktop\\practica-integradora-4-back\\node_modules\\mongoose\\lib\\connection.js:875:11)\n    at NativeConnection.openUri (C:\\Users\\Evelin\\Desktop\\practica-integradora-4-back\\node_modules\\mongoose\\lib\\connection.js:826:11)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)"}
error: Error al conectar a MongoDB: connect ETIMEDOUT 52.67.72.115:27017 {"connectionGeneration":1,"stack":"MongoNetworkError: connect ETIMEDOUT 52.67.72.115:27017\n    at connectionFailureError (C:\\Users\\Evelin\\Desktop\\practica-integradora-4-back\\node_modules\\mongodb\\lib\\cmap\\connect.js:353:20)\n    at TLSSocket.<anonymous> (C:\\Users\\Evelin\\Desktop\\practica-integradora-4-back\\node_modules\\mongodb\\lib\\cmap\\connect.js:268:44)\n    at Object.onceWrapper (node:events:629:26)\n    at TLSSocket.emit (node:events:514:28)\n    at emitErrorNT (node:internal/streams/destroy:151:8)\n    at emitErrorCloseNT (node:internal/streams/destroy:116:3)\n    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)"}
error: Error al conectar a MongoDB: connect ETIMEDOUT 52.67.72.115:27017 {"reason":{"commonWireVersion":0,"compatible":true,"heartbeatFrequencyMS":10000,"localThresholdMS":15,"logicalSessionTimeoutMinutes":30,"maxElectionId":null,"maxSetVersion":null,"servers":{},"setName":"atlas-tea0ch-shard-0","stale":false,"type":"ReplicaSetNoPrimary"},"stack":"MongooseServerSelectionError: connect ETIMEDOUT 52.67.72.115:27017\n    at _handleConnectionErrors (C:\\Users\\Evelin\\Desktop\\practica-integradora-4-back\\node_modules\\mongoose\\lib\\connection.js:875:11)\n    at NativeConnection.openUri (C:\\Users\\Evelin\\Desktop\\practica-integradora-4-back\\node_modules\\mongoose\\lib\\connection.js:826:11)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)"}
error: Error al conectar a MongoDB: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net {"code":"EREFUSED","hostname":"_mongodb._tcp.cluster0.sasvmwp.mongodb.net","stack":"Error: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net\n    at QueryReqWrap.onresolve [as oncomplete] (node:internal/dns/promises:275:17)","syscall":"querySrv"}
error: Error al conectar a MongoDB: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net {"code":"EREFUSED","hostname":"_mongodb._tcp.cluster0.sasvmwp.mongodb.net","stack":"Error: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net\n    at QueryReqWrap.onresolve [as oncomplete] (node:internal/dns/promises:275:17)","syscall":"querySrv"}
error: Error al conectar a MongoDB: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net {"code":"EREFUSED","hostname":"_mongodb._tcp.cluster0.sasvmwp.mongodb.net","stack":"Error: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net\n    at QueryReqWrap.onresolve [as oncomplete] (node:internal/dns/promises:275:17)","syscall":"querySrv"}
error: Error al conectar a MongoDB: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net {"code":"EREFUSED","hostname":"_mongodb._tcp.cluster0.sasvmwp.mongodb.net","stack":"Error: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net\n    at QueryReqWrap.onresolve [as oncomplete] (node:internal/dns/promises:275:17)","syscall":"querySrv"}
error: Error al conectar a MongoDB: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net {"code":"EREFUSED","hostname":"_mongodb._tcp.cluster0.sasvmwp.mongodb.net","stack":"Error: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net\n    at QueryReqWrap.onresolve [as oncomplete] (node:internal/dns/promises:275:17)","syscall":"querySrv"}
error: Error al conectar a MongoDB: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net {"code":"EREFUSED","hostname":"_mongodb._tcp.cluster0.sasvmwp.mongodb.net","stack":"Error: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net\n    at QueryReqWrap.onresolve [as oncomplete] (node:internal/dns/promises:275:17)","syscall":"querySrv"}
error: Error al conectar a MongoDB: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net {"code":"EREFUSED","hostname":"_mongodb._tcp.cluster0.sasvmwp.mongodb.net","stack":"Error: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net\n    at QueryReqWrap.onresolve [as oncomplete] (node:internal/dns/promises:275:17)","syscall":"querySrv"}
error: Error al conectar a MongoDB: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net {"code":"EREFUSED","hostname":"_mongodb._tcp.cluster0.sasvmwp.mongodb.net","stack":"Error: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net\n    at QueryReqWrap.onresolve [as oncomplete] (node:internal/dns/promises:275:17)","syscall":"querySrv"}
error: Error al conectar a MongoDB: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net {"code":"EREFUSED","hostname":"_mongodb._tcp.cluster0.sasvmwp.mongodb.net","stack":"Error: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net\n    at QueryReqWrap.onresolve [as oncomplete] (node:internal/dns/promises:275:17)","syscall":"querySrv"}
error: Error al conectar a MongoDB: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net {"code":"EREFUSED","hostname":"_mongodb._tcp.cluster0.sasvmwp.mongodb.net","stack":"Error: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net\n    at QueryReqWrap.onresolve [as oncomplete] (node:internal/dns/promises:275:17)","syscall":"querySrv"}
error: Error al conectar a MongoDB: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net {"code":"EREFUSED","hostname":"_mongodb._tcp.cluster0.sasvmwp.mongodb.net","stack":"Error: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net\n    at QueryReqWrap.onresolve [as oncomplete] (node:internal/dns/promises:275:17)","syscall":"querySrv"}
error: Error al conectar a MongoDB: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net {"code":"EREFUSED","hostname":"_mongodb._tcp.cluster0.sasvmwp.mongodb.net","stack":"Error: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net\n    at QueryReqWrap.onresolve [as oncomplete] (node:internal/dns/promises:275:17)","syscall":"querySrv"}
error: Error al conectar a MongoDB: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net {"code":"EREFUSED","hostname":"_mongodb._tcp.cluster0.sasvmwp.mongodb.net","stack":"Error: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net\n    at QueryReqWrap.onresolve [as oncomplete] (node:internal/dns/promises:275:17)","syscall":"querySrv"}
error: Error al conectar a MongoDB: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net {"code":"EREFUSED","hostname":"_mongodb._tcp.cluster0.sasvmwp.mongodb.net","stack":"Error: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net\n    at QueryReqWrap.onresolve [as oncomplete] (node:internal/dns/promises:275:17)","syscall":"querySrv"}
error: Error al conectar a MongoDB: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net {"code":"EREFUSED","hostname":"_mongodb._tcp.cluster0.sasvmwp.mongodb.net","stack":"Error: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net\n    at QueryReqWrap.onresolve [as oncomplete] (node:internal/dns/promises:275:17)","syscall":"querySrv"}
error: Error al conectar a MongoDB: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net {"code":"EREFUSED","hostname":"_mongodb._tcp.cluster0.sasvmwp.mongodb.net","stack":"Error: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net\n    at QueryReqWrap.onresolve [as oncomplete] (node:internal/dns/promises:275:17)","syscall":"querySrv"}
error: Error al conectar a MongoDB: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net {"code":"EREFUSED","hostname":"_mongodb._tcp.cluster0.sasvmwp.mongodb.net","stack":"Error: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net\n    at QueryReqWrap.onresolve [as oncomplete] (node:internal/dns/promises:275:17)","syscall":"querySrv"}
error: Error al conectar a MongoDB: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net {"code":"EREFUSED","hostname":"_mongodb._tcp.cluster0.sasvmwp.mongodb.net","stack":"Error: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net\n    at QueryReqWrap.onresolve [as oncomplete] (node:internal/dns/promises:275:17)","syscall":"querySrv"}
error: Error al conectar a MongoDB: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net {"code":"EREFUSED","hostname":"_mongodb._tcp.cluster0.sasvmwp.mongodb.net","stack":"Error: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net\n    at QueryReqWrap.onresolve [as oncomplete] (node:internal/dns/promises:275:17)","syscall":"querySrv"}
error: Error al conectar a MongoDB: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net {"code":"EREFUSED","hostname":"_mongodb._tcp.cluster0.sasvmwp.mongodb.net","stack":"Error: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net\n    at QueryReqWrap.onresolve [as oncomplete] (node:internal/dns/promises:275:17)","syscall":"querySrv"}
error: Error al conectar a MongoDB: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net {"code":"EREFUSED","hostname":"_mongodb._tcp.cluster0.sasvmwp.mongodb.net","stack":"Error: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net\n    at QueryReqWrap.onresolve [as oncomplete] (node:internal/dns/promises:275:17)","syscall":"querySrv"}
error: Error al conectar a MongoDB: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net {"code":"EREFUSED","hostname":"_mongodb._tcp.cluster0.sasvmwp.mongodb.net","stack":"Error: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net\n    at QueryReqWrap.onresolve [as oncomplete] (node:internal/dns/promises:275:17)","syscall":"querySrv"}
error: Error al conectar a MongoDB: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net {"code":"EREFUSED","hostname":"_mongodb._tcp.cluster0.sasvmwp.mongodb.net","stack":"Error: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net\n    at QueryReqWrap.onresolve [as oncomplete] (node:internal/dns/promises:275:17)","syscall":"querySrv"}
error: Error al conectar a MongoDB: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net {"code":"EREFUSED","hostname":"_mongodb._tcp.cluster0.sasvmwp.mongodb.net","stack":"Error: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net\n    at QueryReqWrap.onresolve [as oncomplete] (node:internal/dns/promises:275:17)","syscall":"querySrv"}
error: Error al conectar a MongoDB: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net {"code":"EREFUSED","hostname":"_mongodb._tcp.cluster0.sasvmwp.mongodb.net","stack":"Error: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net\n    at QueryReqWrap.onresolve [as oncomplete] (node:internal/dns/promises:275:17)","syscall":"querySrv"}
error: Error al conectar a MongoDB: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net {"code":"EREFUSED","hostname":"_mongodb._tcp.cluster0.sasvmwp.mongodb.net","stack":"Error: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net\n    at QueryReqWrap.onresolve [as oncomplete] (node:internal/dns/promises:275:17)","syscall":"querySrv"}
error: Error al conectar a MongoDB: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net {"code":"EREFUSED","hostname":"_mongodb._tcp.cluster0.sasvmwp.mongodb.net","stack":"Error: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net\n    at QueryReqWrap.onresolve [as oncomplete] (node:internal/dns/promises:275:17)","syscall":"querySrv"}
error: Error al conectar a MongoDB: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net {"code":"EREFUSED","hostname":"_mongodb._tcp.cluster0.sasvmwp.mongodb.net","stack":"Error: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net\n    at QueryReqWrap.onresolve [as oncomplete] (node:internal/dns/promises:275:17)","syscall":"querySrv"}
error: Error al conectar a MongoDB: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net {"code":"EREFUSED","hostname":"_mongodb._tcp.cluster0.sasvmwp.mongodb.net","stack":"Error: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net\n    at QueryReqWrap.onresolve [as oncomplete] (node:internal/dns/promises:275:17)","syscall":"querySrv"}
error: Error al conectar a MongoDB: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net {"code":"EREFUSED","hostname":"_mongodb._tcp.cluster0.sasvmwp.mongodb.net","stack":"Error: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net\n    at QueryReqWrap.onresolve [as oncomplete] (node:internal/dns/promises:275:17)","syscall":"querySrv"}
error: Error al conectar a MongoDB: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net {"code":"EREFUSED","hostname":"_mongodb._tcp.cluster0.sasvmwp.mongodb.net","stack":"Error: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net\n    at QueryReqWrap.onresolve [as oncomplete] (node:internal/dns/promises:275:17)","syscall":"querySrv"}
error: Error al conectar a MongoDB: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net {"code":"EREFUSED","hostname":"_mongodb._tcp.cluster0.sasvmwp.mongodb.net","stack":"Error: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net\n    at QueryReqWrap.onresolve [as oncomplete] (node:internal/dns/promises:275:17)","syscall":"querySrv"}
error: Error al conectar a MongoDB: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net {"code":"EREFUSED","hostname":"_mongodb._tcp.cluster0.sasvmwp.mongodb.net","stack":"Error: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net\n    at QueryReqWrap.onresolve [as oncomplete] (node:internal/dns/promises:275:17)","syscall":"querySrv"}
error: Error al conectar a MongoDB: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net {"code":"EREFUSED","hostname":"_mongodb._tcp.cluster0.sasvmwp.mongodb.net","stack":"Error: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net\n    at QueryReqWrap.onresolve [as oncomplete] (node:internal/dns/promises:275:17)","syscall":"querySrv"}
error: Error al conectar a MongoDB: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net {"code":"EREFUSED","hostname":"_mongodb._tcp.cluster0.sasvmwp.mongodb.net","stack":"Error: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net\n    at QueryReqWrap.onresolve [as oncomplete] (node:internal/dns/promises:275:17)","syscall":"querySrv"}
error: Error al conectar a MongoDB: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net {"code":"EREFUSED","hostname":"_mongodb._tcp.cluster0.sasvmwp.mongodb.net","stack":"Error: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net\n    at QueryReqWrap.onresolve [as oncomplete] (node:internal/dns/promises:275:17)","syscall":"querySrv"}
error: Error al conectar a MongoDB: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net {"code":"EREFUSED","hostname":"_mongodb._tcp.cluster0.sasvmwp.mongodb.net","stack":"Error: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net\n    at QueryReqWrap.onresolve [as oncomplete] (node:internal/dns/promises:275:17)","syscall":"querySrv"}
error: Error al conectar a MongoDB: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net {"code":"EREFUSED","hostname":"_mongodb._tcp.cluster0.sasvmwp.mongodb.net","stack":"Error: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net\n    at QueryReqWrap.onresolve [as oncomplete] (node:internal/dns/promises:275:17)","syscall":"querySrv"}
error: Error al conectar a MongoDB: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net {"code":"EREFUSED","hostname":"_mongodb._tcp.cluster0.sasvmwp.mongodb.net","stack":"Error: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net\n    at QueryReqWrap.onresolve [as oncomplete] (node:internal/dns/promises:275:17)","syscall":"querySrv"}
error: Error al conectar a MongoDB: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net {"code":"EREFUSED","hostname":"_mongodb._tcp.cluster0.sasvmwp.mongodb.net","stack":"Error: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net\n    at QueryReqWrap.onresolve [as oncomplete] (node:internal/dns/promises:275:17)","syscall":"querySrv"}
error: Error al conectar a MongoDB: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net {"code":"EREFUSED","hostname":"_mongodb._tcp.cluster0.sasvmwp.mongodb.net","stack":"Error: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net\n    at QueryReqWrap.onresolve [as oncomplete] (node:internal/dns/promises:275:17)","syscall":"querySrv"}
error: Error al conectar a MongoDB: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net {"code":"EREFUSED","hostname":"_mongodb._tcp.cluster0.sasvmwp.mongodb.net","stack":"Error: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net\n    at QueryReqWrap.onresolve [as oncomplete] (node:internal/dns/promises:275:17)","syscall":"querySrv"}
error: Error al conectar a MongoDB: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net {"code":"EREFUSED","hostname":"_mongodb._tcp.cluster0.sasvmwp.mongodb.net","stack":"Error: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net\n    at QueryReqWrap.onresolve [as oncomplete] (node:internal/dns/promises:275:17)","syscall":"querySrv"}
error: Error al conectar a MongoDB: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net {"code":"EREFUSED","hostname":"_mongodb._tcp.cluster0.sasvmwp.mongodb.net","stack":"Error: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net\n    at QueryReqWrap.onresolve [as oncomplete] (node:internal/dns/promises:275:17)","syscall":"querySrv"}
error: Error al conectar a MongoDB: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net {"code":"EREFUSED","hostname":"_mongodb._tcp.cluster0.sasvmwp.mongodb.net","stack":"Error: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net\n    at QueryReqWrap.onresolve [as oncomplete] (node:internal/dns/promises:275:17)","syscall":"querySrv"}
error: Error eliminando usuarios inactivos: Cannot read properties of undefined (reading 'status') {"stack":"TypeError: Cannot read properties of undefined (reading 'status')\n    at UserController.cleanInactiveUsers (C:\\Users\\Evelin\\Desktop\\practica-integradora-4-back\\src\\controllers\\user.controllers.js:321:9)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)"}
error: Error al conectar a MongoDB: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net {"code":"EREFUSED","hostname":"_mongodb._tcp.cluster0.sasvmwp.mongodb.net","stack":"Error: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net\n    at QueryReqWrap.onresolve [as oncomplete] (node:internal/dns/promises:275:17)","syscall":"querySrv"}
error: Error al conectar a MongoDB: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net {"code":"EREFUSED","hostname":"_mongodb._tcp.cluster0.sasvmwp.mongodb.net","stack":"Error: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net\n    at QueryReqWrap.onresolve [as oncomplete] (node:internal/dns/promises:275:17)","syscall":"querySrv"}
error: Error al conectar a MongoDB: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net {"code":"EREFUSED","hostname":"_mongodb._tcp.cluster0.sasvmwp.mongodb.net","stack":"Error: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net\n    at QueryReqWrap.onresolve [as oncomplete] (node:internal/dns/promises:275:17)","syscall":"querySrv"}
error: Error al conectar a MongoDB: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net {"code":"EREFUSED","hostname":"_mongodb._tcp.cluster0.sasvmwp.mongodb.net","stack":"Error: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net\n    at QueryReqWrap.onresolve [as oncomplete] (node:internal/dns/promises:275:17)","syscall":"querySrv"}
error: Error en el registro de usuario: Error al intentar crear un usuario {"causa":" Los datos estan incompletos o no son válidos. \n    Necesitamos recibir los siguientes datos: \n    - Nombre: String, pero recibimos undefined\n    - Apellido: String, pero recibimos undefined\n    - Email: String, recibimos undefined\n    -Password: recibimos undefined\n    -Edad: recibimos undefined\n    ","code":2,"name":"Error","stack":"Error: Error al intentar crear un usuario\n    at CustomErrors.createError (C:\\Users\\Evelin\\Desktop\\practica-integradora-4-back\\src\\services\\errors\\custom-error.js:3:23)\n    at register (C:\\Users\\Evelin\\Desktop\\practica-integradora-4-back\\src\\controllers\\user.controllers.js:25:28)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)"}
error: Error en el registro de usuario: Error al intentar crear un usuario {"causa":" Los datos estan incompletos o no son válidos. \n    Necesitamos recibir los siguientes datos: \n    - Nombre: String, pero recibimos undefined\n    - Apellido: String, pero recibimos undefined\n    - Email: String, recibimos undefined\n    -Password: recibimos undefined\n    -Edad: recibimos undefined\n    ","code":2,"name":"Error","stack":"Error: Error al intentar crear un usuario\n    at CustomErrors.createError (C:\\Users\\Evelin\\Desktop\\practica-integradora-4-back\\src\\services\\errors\\custom-error.js:3:23)\n    at register (C:\\Users\\Evelin\\Desktop\\practica-integradora-4-back\\src\\controllers\\user.controllers.js:25:28)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)"}
error: Error al conectar a MongoDB: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net {"code":"EREFUSED","hostname":"_mongodb._tcp.cluster0.sasvmwp.mongodb.net","stack":"Error: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net\n    at QueryReqWrap.onresolve [as oncomplete] (node:internal/dns/promises:275:17)","syscall":"querySrv"}
error: Error eliminando usuarios inactivos: Cannot read properties of undefined (reading 'status') {"stack":"TypeError: Cannot read properties of undefined (reading 'status')\n    at UserController.cleanInactiveUsers (C:\\Users\\Evelin\\Desktop\\practica-integradora-4-back\\src\\controllers\\user.controllers.js:326:9)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)"}
error: Error al conectar a MongoDB: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net {"code":"EREFUSED","hostname":"_mongodb._tcp.cluster0.sasvmwp.mongodb.net","stack":"Error: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net\n    at QueryReqWrap.onresolve [as oncomplete] (node:internal/dns/promises:275:17)","syscall":"querySrv"}
error: Error al conectar a MongoDB: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net {"code":"EREFUSED","hostname":"_mongodb._tcp.cluster0.sasvmwp.mongodb.net","stack":"Error: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net\n    at QueryReqWrap.onresolve [as oncomplete] (node:internal/dns/promises:275:17)","syscall":"querySrv"}
error: Error al conectar a MongoDB: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net {"code":"EREFUSED","hostname":"_mongodb._tcp.cluster0.sasvmwp.mongodb.net","stack":"Error: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net\n    at QueryReqWrap.onresolve [as oncomplete] (node:internal/dns/promises:275:17)","syscall":"querySrv"}
error: Error al conectar a MongoDB: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net {"code":"EREFUSED","hostname":"_mongodb._tcp.cluster0.sasvmwp.mongodb.net","stack":"Error: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net\n    at QueryReqWrap.onresolve [as oncomplete] (node:internal/dns/promises:275:17)","syscall":"querySrv"}
error: Error al conectar a MongoDB: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net {"code":"EREFUSED","hostname":"_mongodb._tcp.cluster0.sasvmwp.mongodb.net","stack":"Error: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net\n    at QueryReqWrap.onresolve [as oncomplete] (node:internal/dns/promises:275:17)","syscall":"querySrv"}
error: Error al conectar a MongoDB: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net {"code":"EREFUSED","hostname":"_mongodb._tcp.cluster0.sasvmwp.mongodb.net","stack":"Error: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net\n    at QueryReqWrap.onresolve [as oncomplete] (node:internal/dns/promises:275:17)","syscall":"querySrv"}
error: Error al conectar a MongoDB: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net {"code":"EREFUSED","hostname":"_mongodb._tcp.cluster0.sasvmwp.mongodb.net","stack":"Error: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net\n    at QueryReqWrap.onresolve [as oncomplete] (node:internal/dns/promises:275:17)","syscall":"querySrv"}
error: Error al conectar a MongoDB: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net {"code":"EREFUSED","hostname":"_mongodb._tcp.cluster0.sasvmwp.mongodb.net","stack":"Error: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net\n    at QueryReqWrap.onresolve [as oncomplete] (node:internal/dns/promises:275:17)","syscall":"querySrv"}
error: Error al conectar a MongoDB: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net {"code":"EREFUSED","hostname":"_mongodb._tcp.cluster0.sasvmwp.mongodb.net","stack":"Error: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net\n    at QueryReqWrap.onresolve [as oncomplete] (node:internal/dns/promises:275:17)","syscall":"querySrv"}
error: Error al conectar a MongoDB: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net {"code":"EREFUSED","hostname":"_mongodb._tcp.cluster0.sasvmwp.mongodb.net","stack":"Error: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net\n    at QueryReqWrap.onresolve [as oncomplete] (node:internal/dns/promises:275:17)","syscall":"querySrv"}
error: Error al conectar a MongoDB: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net {"code":"EREFUSED","hostname":"_mongodb._tcp.cluster0.sasvmwp.mongodb.net","stack":"Error: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net\n    at QueryReqWrap.onresolve [as oncomplete] (node:internal/dns/promises:275:17)","syscall":"querySrv"}
error: Error al conectar a MongoDB: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net {"code":"EREFUSED","hostname":"_mongodb._tcp.cluster0.sasvmwp.mongodb.net","stack":"Error: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net\n    at QueryReqWrap.onresolve [as oncomplete] (node:internal/dns/promises:275:17)","syscall":"querySrv"}
error: Error al conectar a MongoDB: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net {"code":"EREFUSED","hostname":"_mongodb._tcp.cluster0.sasvmwp.mongodb.net","stack":"Error: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net\n    at QueryReqWrap.onresolve [as oncomplete] (node:internal/dns/promises:275:17)","syscall":"querySrv"}
error: Error al conectar a MongoDB: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net {"code":"EREFUSED","hostname":"_mongodb._tcp.cluster0.sasvmwp.mongodb.net","stack":"Error: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net\n    at QueryReqWrap.onresolve [as oncomplete] (node:internal/dns/promises:275:17)","syscall":"querySrv"}
error: Error al conectar a MongoDB: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net {"code":"EREFUSED","hostname":"_mongodb._tcp.cluster0.sasvmwp.mongodb.net","stack":"Error: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net\n    at QueryReqWrap.onresolve [as oncomplete] (node:internal/dns/promises:275:17)","syscall":"querySrv"}
error: Error al conectar a MongoDB: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net {"code":"EREFUSED","hostname":"_mongodb._tcp.cluster0.sasvmwp.mongodb.net","stack":"Error: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net\n    at QueryReqWrap.onresolve [as oncomplete] (node:internal/dns/promises:275:17)","syscall":"querySrv"}
error: Error al conectar a MongoDB: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net {"code":"EREFUSED","hostname":"_mongodb._tcp.cluster0.sasvmwp.mongodb.net","stack":"Error: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net\n    at QueryReqWrap.onresolve [as oncomplete] (node:internal/dns/promises:275:17)","syscall":"querySrv"}
error: Error al conectar a MongoDB: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net {"code":"EREFUSED","hostname":"_mongodb._tcp.cluster0.sasvmwp.mongodb.net","stack":"Error: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net\n    at QueryReqWrap.onresolve [as oncomplete] (node:internal/dns/promises:275:17)","syscall":"querySrv"}
error: Error al conectar a MongoDB: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net {"code":"EREFUSED","hostname":"_mongodb._tcp.cluster0.sasvmwp.mongodb.net","stack":"Error: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net\n    at QueryReqWrap.onresolve [as oncomplete] (node:internal/dns/promises:275:17)","syscall":"querySrv"}
error: Error al conectar a MongoDB: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net {"code":"EREFUSED","hostname":"_mongodb._tcp.cluster0.sasvmwp.mongodb.net","stack":"Error: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net\n    at QueryReqWrap.onresolve [as oncomplete] (node:internal/dns/promises:275:17)","syscall":"querySrv"}
error: Error al conectar a MongoDB: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net {"code":"EREFUSED","hostname":"_mongodb._tcp.cluster0.sasvmwp.mongodb.net","stack":"Error: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net\n    at QueryReqWrap.onresolve [as oncomplete] (node:internal/dns/promises:275:17)","syscall":"querySrv"}
error: Error al conectar a MongoDB: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net {"code":"EREFUSED","hostname":"_mongodb._tcp.cluster0.sasvmwp.mongodb.net","stack":"Error: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net\n    at QueryReqWrap.onresolve [as oncomplete] (node:internal/dns/promises:275:17)","syscall":"querySrv"}
error: Error al conectar a MongoDB: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net {"code":"EREFUSED","hostname":"_mongodb._tcp.cluster0.sasvmwp.mongodb.net","stack":"Error: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net\n    at QueryReqWrap.onresolve [as oncomplete] (node:internal/dns/promises:275:17)","syscall":"querySrv"}
error: Error al conectar a MongoDB: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net {"code":"EREFUSED","hostname":"_mongodb._tcp.cluster0.sasvmwp.mongodb.net","stack":"Error: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net\n    at QueryReqWrap.onresolve [as oncomplete] (node:internal/dns/promises:275:17)","syscall":"querySrv"}
error: Error eliminando usuarios inactivos: Cannot read properties of undefined (reading 'status') {"stack":"TypeError: Cannot read properties of undefined (reading 'status')\n    at UserController.cleanInactiveUsers (C:\\Users\\Evelin\\Desktop\\practica-integradora-4-back\\src\\controllers\\user.controllers.js:326:9)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)"}
error: Error al conectar a MongoDB: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net {"code":"EREFUSED","hostname":"_mongodb._tcp.cluster0.sasvmwp.mongodb.net","stack":"Error: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net\n    at QueryReqWrap.onresolve [as oncomplete] (node:internal/dns/promises:275:17)","syscall":"querySrv"}
error: Error al conectar a MongoDB: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net {"code":"EREFUSED","hostname":"_mongodb._tcp.cluster0.sasvmwp.mongodb.net","stack":"Error: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net\n    at QueryReqWrap.onresolve [as oncomplete] (node:internal/dns/promises:275:17)","syscall":"querySrv"}
error: Error al conectar a MongoDB: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net {"code":"EREFUSED","hostname":"_mongodb._tcp.cluster0.sasvmwp.mongodb.net","stack":"Error: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net\n    at QueryReqWrap.onresolve [as oncomplete] (node:internal/dns/promises:275:17)","syscall":"querySrv"}
error: Error al conectar a MongoDB: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net {"code":"EREFUSED","hostname":"_mongodb._tcp.cluster0.sasvmwp.mongodb.net","stack":"Error: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net\n    at QueryReqWrap.onresolve [as oncomplete] (node:internal/dns/promises:275:17)","syscall":"querySrv"}
error: Error al conectar a MongoDB: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net {"code":"EREFUSED","hostname":"_mongodb._tcp.cluster0.sasvmwp.mongodb.net","stack":"Error: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net\n    at QueryReqWrap.onresolve [as oncomplete] (node:internal/dns/promises:275:17)","syscall":"querySrv"}
error: Error al conectar a MongoDB: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net {"code":"EREFUSED","hostname":"_mongodb._tcp.cluster0.sasvmwp.mongodb.net","stack":"Error: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net\n    at QueryReqWrap.onresolve [as oncomplete] (node:internal/dns/promises:275:17)","syscall":"querySrv"}
error: Error al conectar a MongoDB: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net {"code":"EREFUSED","hostname":"_mongodb._tcp.cluster0.sasvmwp.mongodb.net","stack":"Error: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net\n    at QueryReqWrap.onresolve [as oncomplete] (node:internal/dns/promises:275:17)","syscall":"querySrv"}
error: Error al conectar a MongoDB: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net {"code":"EREFUSED","hostname":"_mongodb._tcp.cluster0.sasvmwp.mongodb.net","stack":"Error: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net\n    at QueryReqWrap.onresolve [as oncomplete] (node:internal/dns/promises:275:17)","syscall":"querySrv"}
error: Error eliminando usuarios inactivos: Operation `users.find()` buffering timed out after 10000ms {"stack":"MongooseError: Operation `users.find()` buffering timed out after 10000ms\n    at Timeout.<anonymous> (C:\\Users\\Evelin\\Desktop\\practica-integradora-4-back\\node_modules\\mongoose\\lib\\drivers\\node-mongodb-native\\collection.js:185:23)\n    at listOnTimeout (node:internal/timers:573:17)\n    at process.processTimers (node:internal/timers:514:7)"}
error: Error al conectar a MongoDB: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net {"code":"EREFUSED","hostname":"_mongodb._tcp.cluster0.sasvmwp.mongodb.net","stack":"Error: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net\n    at QueryReqWrap.onresolve [as oncomplete] (node:internal/dns/promises:275:17)","syscall":"querySrv"}
error: undefined
error: undefined
error: undefined
error: undefined
error: undefined
error: undefined
error: undefined
error: Error al conectar a MongoDB: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net {"code":"EREFUSED","hostname":"_mongodb._tcp.cluster0.sasvmwp.mongodb.net","stack":"Error: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net\n    at QueryReqWrap.onresolve [as oncomplete] (node:internal/dns/promises:275:17)","syscall":"querySrv"}
error: Error al conectar a MongoDB: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net {"code":"EREFUSED","hostname":"_mongodb._tcp.cluster0.sasvmwp.mongodb.net","stack":"Error: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net\n    at QueryReqWrap.onresolve [as oncomplete] (node:internal/dns/promises:275:17)","syscall":"querySrv"}
error: undefined
error: undefined
error: Error al conectar a MongoDB: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net {"code":"EREFUSED","hostname":"_mongodb._tcp.cluster0.sasvmwp.mongodb.net","stack":"Error: querySrv EREFUSED _mongodb._tcp.cluster0.sasvmwp.mongodb.net\n    at QueryReqWrap.onresolve [as oncomplete] (node:internal/dns/promises:275:17)","syscall":"querySrv"}
